src/sentry/sentry_apps/api/endpoints/sentry_app_publish_request.py

```diff
@@ -58,8 +58,8 @@ def post(self, request: Request, sentry_app) -> Response:
                 status=400,
             )
 
-        assert isinstance(request.user, User) or isinstance(
-            request.user, RpcUser
+        assert isinstance(
+            request.user, (User, RpcUser)
```

Comment:
similarly, mypy now knows this fact statically (but is not smart enough to see the `isinstance(..., (several, types))`!)